README.md

```diff
@@ -99,6 +99,17 @@ class { '::heka':
 }
 ```
 
+####Package versioning
+
+To specify a particular Heka package version to install, use the `package_download_url` and `version` parameters:
```

Comment:
Do you want to include a bit in there about downgrades? I remember you mentioned the rpm provider didn't quite do those.